Philips.CodeAnalysis.Common/SingleDiagnosticAnalyzer.cs

```diff
@@ -17,7 +17,8 @@ public abstract class SingleDiagnosticAnalyzer : DiagnosticAnalyzer
 		{
```

Comment:
Can we remove helpUri from the constructor? Or maybe that comes later?

Reply:
Later, there are several Analyzers that use it to share a useful link.